concordia/templates/registration/activation_email.txt

```diff
@@ -4,7 +4,7 @@ Hi {{ user }},
 Thank you for joining Make Your Mark! Please verify your email address in the next {{ expiration_days |multiply:24 }} hours.
 
 To complete activation, please click the following link:
-<button class=button><a href='http://{{ site }}{% url "registration_activate" activation_key %}'>Verify Now</a></button>
+http://{{ site }}{% url "registration_activate" activation_key %}
```

Comment:
This should be `https` for production. I'm inclined to say we do that now and either get ACM setup on our load-balancers or get some LC internal certs for every other tier.

Reply:
Changed to https, although the load balancers still need to catch up